src/Api/api.js

```diff
@@ -1,4 +1,4 @@
-const BASE_URL = "https://statefree-redilab.onrender.com";
+const BASE_URL = process.env.REACT_APP_BASE_URL;
```

Comment:
Hi Maryam! I think that without github workflows set up for a production environment, this will only work when running the application locally.

What you can do is: 

`const BASE_URL = process.env.REACT_APP_BASE_URL ?? "https://statefree-redilab.onrender.com";`

This will 1) use your local environment variable (e.g. `localhost:8000`) if it is present, and otherwise fall back to the production endpoint.

Reply:
I have deleted .env and I use just hardcode :)

Reply:
Hi @shabanzadeh , I think this didn't work here - I think you didn't push the change

Please make sure to ask for a re-review before merging the PR next time :))